x-pack/plugins/canvas/server/feature.test.ts

```diff
@@ -6,10 +6,10 @@
  */
 
 import { getCanvasFeature } from './feature';
-import { ReportingStart } from '@kbn/reporting-plugin/server/types';
```

Comment:
I think this it was more preferable to get this directly from the reporting plugin. It is something that consumers need and that is the place that it would intuitively be exported from.

Sorry, but I'd like you to revert the changes that switched importing `ReportingStart` from `@kbn/reporting-plugin/server`. 

Internally, we can't import `ReportingStart` into any package code where it might be needed. Let's keep `CommonReportingSetup` as you have it, but use it internally by making sure to only import it from package code.

Reply:
Ok I think I am following - changed in commit [bd80d0e](https://github.com/elastic/kibana/pull/162845/commits/bd80d0ea1d79005e035fe6cd5995088dd4d22bcf) thanks!